docker_dev/mysql/01-setup.sql

```diff
@@ -9,5 +9,4 @@ GRANT CREATE ON *.* TO 'pennydreadful'@'%';
 
 CREATE DATABASE decksite;
 CREATE DATABASE cards;
-CREATE DATABASE pdlogs;
```

Comment:
creating an empty db here causes the check in logsite to skip all the migrations: https://github.com/PennyDreadfulMTG/Penny-Dreadful-Tools/blob/master/logsite/__init__.py#L26